src/app/search/search.component.html

```diff
@@ -0,0 +1,3 @@
+
+ 
+  <button class="btn btn-success" (click)="onNextPage()">Next Page</button>
```

Comment:
use variables for constants

Reply:
I created const file and declared variable in that file.